config/rbac/operand_role.yaml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: agent
+rules:
+- apiGroups:
+  - "security.openshift.io"
+  resources:
+  - securitycontextconstraints
+  resourceNames:
+  - node-observability-scc
```

Comment:
Would be good to remove the trailing `-scc` from the SCC name as it's redundant.